api/src/main/java/org/pmiops/workbench/db/model/DbCdrVersion.java

```diff
@@ -208,7 +210,7 @@ public DbCdrVersion setWgsFilterSetName(String wgsFilterSetName) {
 
   @Column(name = "has_fitbit_data")
   public Boolean getHasFitbitData() {
-    return hasFitbitData == null ? false : hasFitbitData;
+    return hasFitbitData != null && hasFitbitData;
```

Comment:
we often use `Boolean.TRUE.equals(foo)` to translate nullable-Booleans to primitive ones. 
 This converts nulls to false, like what we want here.

Reply:
I like that. I converted them.